src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md

```diff
@@ -53,10 +53,10 @@ Complete the following steps to install a Galasa Ecosystem by using Helm:
         helm repo add galasa https://galasa-dev.github.io/helm
         ```
     - If the repository exists, run the ```helm repo update``` command to get the latest versions of the packages and then run ```helm search repo galasa``` to see the available charts.<br>
-    _Note:_ The Galasa Ecosystem Helm chart deploys three persistent volumes (PVs). If you need to provide a Kubernetes storage class for these PVs, download the  <a href= https://github.com/galasa-dev/helm/blob/ecosystem-0.34.0/charts/ecosystem/values.yaml target="_blank"> values.yaml</a> file and update the `storageClass` value in the file with the name of a valid storage class on your cluster. If you are deploying to minikube, you can optionally use the standard storage class that is created for you by minikube, but this is not required.
-1. Download the <a href= https://github.com/galasa-dev/helm/blob/ecosystem-0.34.0/charts/ecosystem/values.yaml target="_blank"> values.yaml</a> file if you have not done so already, and edit the values of the following properties: 
+    _Note:_ The Galasa Ecosystem Helm chart deploys three persistent volumes (PVs). If you need to provide a Kubernetes storage class for these PVs, download the  <a href= https://github.com/galasa-dev/helm/blob/main/charts/ecosystem/values.yaml target="_blank"> values.yaml</a> file and update the `storageClass` value in the file with the name of a valid storage class on your cluster. If you are deploying to minikube, you can optionally use the standard storage class that is created for you by minikube, but this is not required. 
+1. Download the <a href= https://github.com/galasa-dev/helm/blob/main/charts/ecosystem/values.yaml target="_blank"> values.yaml</a> file if you have not done so already, and edit the values of the following properties: 
```

Comment:
We'll want to keep these links pointed at the latest release instead of main, since main can have changes/values that aren't released yet

Reply:
Will change back once 0.34.0 has released.